tfhe/docs/getting_started/quick_start.md

```diff
@@ -2,11 +2,81 @@
 
 This document explains the basic steps of using the high-level API of **TFHE-rs.**
 
-## Workflow explanation
+## Setting up a Rust project
 
-These are the steps to use the **TFHE-rs** high-level API:
+If you already know how to set-up a Rust project you can go to the next [section](#using-tfhe-rs-and-its-apis).
 
-1. [Import the **TFHE-rs** prelude](quick\_start.md#imports)
+As a first step you need to install the Rust programming language tools. You can do so by visiting https://rustup.rs/ and following instructions there. If you want alternative ways of installing Rust you can check out https://rust-lang.github.io/rustup/installation/other.html for more options.
+
+Once you have Rust installed you should be able to call the build and package manager `Cargo`:
```

Comment:
First, install the Rust programming language tools. Visit https://rustup.rs/ and follow the instructions. For alternative installation methods, refer to the [official Rust installation page](https://rust-lang.github.io/rustup/installation/other.html).

After installing Rust, you can call the build and package manager `Cargo`: